rosidl_generator_py/CMakeLists.txt

```diff
@@ -4,6 +4,8 @@ project(rosidl_generator_py)
 
 find_package(ament_cmake REQUIRED)
 find_package(ament_cmake_python REQUIRED)
+find_package(rosidl_cmake REQUIRED)
+find_package(rosidl_generator_c REQUIRED)
```

Comment:
Yes, test dependencies are only available when testing is enabled therefore can only be used within the conditional block. Then these are also no build dependencies anymore.